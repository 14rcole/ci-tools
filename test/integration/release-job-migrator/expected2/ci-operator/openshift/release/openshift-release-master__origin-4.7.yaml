releases:
  initial:
    candidate:
      product: ocp
      relative: 1
      stream: ci
      version: "4.7"
  latest:
    candidate:
      product: ocp
      stream: ci
      version: "4.7"
resources:
  '*':
    requests:
      cpu: 100m
      memory: 200Mi
tests:
- as: e2e-aws
  interval: 48h
  steps:
    cluster_profile: aws
    workflow: openshift-e2e-aws
zz_generated_metadata:
  branch: master
  org: openshift
  repo: release
  variant: origin-4.7
