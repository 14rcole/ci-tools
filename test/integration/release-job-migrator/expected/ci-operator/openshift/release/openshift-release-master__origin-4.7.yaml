base_images:
  base:
    name: "4.7"
    namespace: ocp
    tag: base
releases:
  latest:
    candidate:
      product: ocp
      stream: ci
      version: "4.7"
resources:
  '*':
    requests:
      cpu: 100m
      memory: 200Mi
tests:
- as: e2e-aws
  cron: '@every 48h'
  steps:
    cluster_profile: aws
    workflow: openshift-e2e-aws
- as: e2e-gcp-upgrade
  cron: '@yearly'
  steps:
    cluster_profile: gcp
    workflow: openshift-upgrade-gcp-loki
zz_generated_metadata:
  branch: master
  org: openshift
  repo: release
  variant: origin-4.7
